agent-install/agent-uninstall.sh

```diff
@@ -267,15 +271,45 @@ function deleteAgentResources() {
 
     set +e
     log_info "Deleting agent deployment..."
-    $KUBECTL delete deployment $DEPLOYMENT_NAME -n $AGENT_NAMESPACE --force=true --grace-period=0
+    
+    if [ "$USE_DELETE_FORCE" != true ]; then
+        $KUBECTL delete deployment $DEPLOYMENT_NAME -n $AGENT_NAMESPACE --grace-period=10
```

Comment:
Please make the timeout (10 in this case) configurable via an env var `DELETE_TIMEOUT`